test/java/seedu/addressbook/storage/StorageFileTest.java

```diff
@@ -112,12 +112,12 @@ private AddressBook getTestAddressBook() throws Exception {
         ab.addPerson(new Person(new Name("John Doe"),
                                 new Phone("98765432", false),
                                 new Email("johnd@gmail.com", false),
-                                new Address("John street, block 123, #01-01", false),
+                                new Address("John street, block 123, #01-01, 12312", false),
```

Comment:
Your postal code format is a little inconsistent - some include the hex signs, but others don't. You might want to consider keeping it consistent next time :)